Sources/Plasma/FeatureLib/pfPython/cyPythonInterface.cpp

```diff
@@ -1447,7 +1447,7 @@ PyObject* PythonInterface::CreateModule(const char* module)
     {
         // clear it
         hsAssert(false, ST::format("ERROR! Creating a python module of the same name - {}", module).c_str());
-        _PyModule_Clear(m);
+        //_PyModule_Clear(m);
```

Comment:
At some point we need to actually address this (see https://github.com/H-uru/Plasma/issues/1649)